qiskit/compiler/transpiler.py

```diff
@@ -660,6 +668,12 @@ def _parse_transpile_args(
     }
 
     list_transpile_args = []
+    if not ignore_backend_default_methods:
+        if scheduling_method is None and hasattr(backend, "get_scheduling_stage_method"):
```

Comment:
This seems to me like restricting the flexibility of `StagedPassManager` where we can define transpiration with arbitrary combination of passes, i.e. the class can take `stages` argument. I have no idea how current configuration of passes is generic, but alternatively backend can provide full set of staged passes rather than providing a particular subset passes that Qiskit transpiler defines. For example, some ecosystem software may offer a service that overrides an IBM backend with fancy pass manager that makes circuit more robust to noise -- I also think we no longer say this is a plugin. If this means to be a plugin, perhaps current implementation is fine.

Reply:
Okey, so we think current default stages of qiskit transpiler is sort of stable in future. My only concern was the stability of new API. To me current mechanism based off of #8305 seems reasonable.